api/api.proto

```diff
@@ -36,6 +36,21 @@ message DeleteClientResp {
   bool not_found = 1;
 }
 
+// UpdateClientReq is a request to update an exisitng client.
+message UpdateClientReq {
+    string id = 1;
+    repeated string redirect_uris = 2;
+    repeated string trusted_peers = 3;
+    bool public = 4;
```

Comment:
I think we should either just overwrite the _public_ field with whatever was in the `UpdateClientReq` (what you had pushed before; sorry about that) -- and **ensure that documentation** makes it obvious that the behaviour isn't _patchy_ when it comes to `public`; **or** keep the code as-is and remove the field from this message.

Reply:
I would prefer to remove the `public` field since is not used. Please could you point me to the documentation, which should I update?

Reply:
The filed is removed.